src/ua-parser.js

```diff
@@ -90,7 +91,7 @@
     },
     trim = function (str, len) {
       if (typeof str === STR_TYPE) {
-        str = str.replace(/^\s\s*/, EMPTY).replace(/\s\s*$/, EMPTY);
+        str = str.replace(/^\s\s*/, EMPTY);
```

Comment:
Thank you for supporting Amplitude! This change will keep the ending spaces, example:
```
"  abc  ".replace(/^\s\s*/, "").replace(/\s\s*$/, "")
// 'abc'
"  abc  ".replace(/^\s\s*/, "")
// 'abc  '
```
Not sure why, so double check here, is this change expected?

Reply:
Never mind, found the change for this with reason: https://github.com/faisalman/ua-parser-js/commit/a6140a17dd0300a35cfc9cff999545f267889411